openfisca_us/variables/gov/irs/credits/cdcc/cdcc_relevant_expenses.py

```diff
@@ -0,0 +1,32 @@
+from openfisca_us.model_api import *
+
+
+class cdcc_relevant_expenses(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "CDCC-relevant care expenses"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/21#c"
+
+    def formula(tax_unit, period, parameters):
+        # First, cap based on the number of eligible care receivers
+        expenses = tax_unit("tax_unit_childcare_expenses", period)
+        max_expenses = parameters(period).irs.credits.cdcc.max
+        count_eligible = tax_unit("count_cdcc_eligible", period)
+        eligible_capped_expenses = min_(
+            expenses, max_expenses * count_eligible
+        )
+        # Then, cap further to the lowest earnings between the taxpayer and spouse
+        earned_income = tax_unit.members("earned", period)
```

Comment:
```suggestion
        person = tax_unit.members
        earned_income = person("earned", period)
```
and so on since it's used elsewhere.